queue_and_stack/dll_stack.py

```diff
@@ -1,18 +1,175 @@
 import sys
 sys.path.append('../doubly_linked_list')
-from doubly_linked_list import DoublyLinkedList
 
 class Stack:
   def __init__(self):
     self.size = 0
     # Why is our DLL a good choice to store our elements?
-    self.storage = 
+    self.storage = DoublyLinkedList()
 
   def push(self, value):
-    pass
+    self.storage.move_to_end(value)
   
   def pop(self):
-    pass
+    self.storage.delete(-1)
```

Comment:
most pop methods return the "popped" value. I'm not seeing that in the spec though, so can't really hold it against you...